tools/src/main/java/com/nedap/archie/creation/RMObjectCreator.java

```diff
@@ -52,9 +52,9 @@ public void set(Object object, String rmAttributeName, List<Object> values) {
 
             Type type = attributeInfo.getType();
             if(type instanceof Class) {
-                Class clazz = (Class) type;
+                Class<?> clazz = (Class<?>) type;
                 if(attributeInfo.isMultipleValued()) {
-                    Collection collection = (Collection) newInstance(attributeInfo);
+                    Collection<Object> collection = (Collection<Object>) newInstance(attributeInfo);
```

Comment:
Should Object be `?` instead here? Or does that not work with casting it?

Reply:
With `Collection<?> collection = (Collection<?>) newInstance(attributeInfo);` you cannot call `collection.addAll(values)` below.